### ISSUE TITLE
support Treesitter highlight groups

### ISSUE THREAD

**Issue by ilAYAli** [romgrk/github-light.vim#3]
Support for Treesitter highlight groups would be much appreciated:

```
TSError
TSPunctDelimiter
TSPunctBracket
TSPunctSpecial
TSConstant
TSConstBuiltin
TSConstMacro
TSStringRegex
TSString
TSStringEscape
TSCharacter
TSNumber
TSBoolean
TSFloat
TSAnnotation
TSAttribute
TSNamespace
TSFuncBuiltin
TSFunction
TSFuncMacro
TSParameter
TSParameterReference
TSMethod
TSField
TSProperty
TSConstructor
TSConditional
TSRepeat
TSLabel
TSKeyword
TSKeywordFunction
TSKeywordOperator
TSOperator
TSException
TSType
TSTypeBuiltin
TSStructure
TSInclude
TSVariable
TSVariableBuiltin
TSText
TSStrong
TSEmphasis
TSUnderline
TSTitle
TSLiteral
TSURI
TSTag
TSTagDelimiter
TSNone
TSQueryLinterError
```

**Comment by romgrk** [comment]
Doesn't treesitter fallbacks on standard groups?

Also note that I'm not using this colorscheme at the moment so I'm unlikely to implement this. I might do it if I come back to this colorscheme but for now I'm with the dark side. I'd accept a PR though.

**Comment by ilAYAli** [comment]
@romgrk yes, treesitter has fallback to standard groups, but additional highlight groups makes for a much better experience.
I am usually on the darker side too, but this is my preferred theme; outside, on a bright summer day, with a beer in my hand, and laptop on my ....lap. Sadly, that is a few months into the future.

**Comment by romgrk** [comment]
I totally understand it, it's also my favourite light theme.

I will put it in my todo list and will try to get it done by this summer. I'll probably be updating [my dark theme](https://github.com/romgrk/doom-one.vim) at the same time, both themes are forks of each other so adding the groups to one makes it trivial to add them to the other.
